gfw_pixetl/sources.py

```diff
@@ -188,7 +188,9 @@ def fetch_meta(self) -> Tuple[BoundingBox, Dict[str, Any]]:
         LOGGER.debug(f"Fetch metadata data for file {self.url} if exists")
 
         try:
-            with rasterio.Env(**GDAL_ENV), rasterio.open(self.url) as src:
+            with rasterio.Env(**GDAL_ENV), rasterio.open(
+                self.url, "r", sharing=False
```

Comment:
This is already the default. I would keep this as is to keep things simple
https://github.com/mapbox/rasterio/blob/master/rasterio/__init__.py#L60